cqrs-user-query-service/src/main/java/dev/amir/userquery/framework/output/adapter/UserOutputAdapter.java

```diff
@@ -27,11 +30,13 @@ public Collection<User> getAll() {
 
     @Override
     public Optional<User> getById(String userId) {
+        log.info("Getting user by ID: {} from the database.");
         return userMongoRepository.findById(userId).map(userMongoMapper::convert);
     }
 
     @Override
     public User save(User user) {
+        log.info("Saving user in the database: {}.");
```

Comment:
the message will print a parameter but the log.info method did not receive a parameter to be printed

{}  this will be replaced by a parameter